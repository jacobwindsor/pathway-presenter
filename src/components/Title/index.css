.slide-title {
    position: absolute;
    top:0;
    width:100%;
    z-index: 10;
    background-color: #80deea;
    box-shadow: rgba(0, 0, 0, 0.117647) 0 1px 6px, rgba(0, 0, 0, 0.117647) 0 1px 4px;
}
.slide-title h1 {
    text-align: center;
}