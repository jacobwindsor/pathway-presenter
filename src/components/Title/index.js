import React from 'react';
import './index.css';

export default (props) => {
    return (
        <div className="slide-title">
            <h1>{props.title}</h1>
        </div>
    )
}