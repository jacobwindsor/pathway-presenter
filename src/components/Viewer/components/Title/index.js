import React from 'react';
import './index.css';

export default (props) => {
    return <h1>{props.title}</h1>
}