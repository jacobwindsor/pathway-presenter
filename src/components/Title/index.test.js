import React from 'react';
import { shallow } from 'enzyme';
import Title from './index';
import toJson from 'enzyme-to-json';

it('should render without failing', () => {
  const comp = shallow(<Title />);
  expect(toJson(comp)).toMatchSnapshot();
});

it('Should show the title', () => {
  const comp = shallow(<Title title="My title" />);
  expect(comp.find('h1')).toHaveLength(1);
  expect(comp.first('h1').contains('My title')).toBeTruthy();
});
