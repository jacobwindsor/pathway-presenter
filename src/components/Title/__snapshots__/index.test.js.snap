// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without failing 1`] = `
<div
  className="slide-title"
>
  <h1 />
</div>
`;
